bugbot/rules/close_intermittents.py

```diff
@@ -17,7 +17,7 @@ def description(self):
         return "Intermittent test failure bugs unchanged in 21 days"
 
     def get_max_actions(self):
-        limit_per_run = 140
+        limit_per_run = 2100
```

Comment:
this 2100 limit, will bugzilla handle that well?  Maybe we keep it <900 which is a large boost.

Reply:
It is only a theoretical limit, because the real limit is the per-team one.